frontend/src/containers/HomePage.js

```diff
@@ -143,7 +183,7 @@ export class HomePage extends Component {
               </div>
             </div>
           </div>
-          <a href='#'>Learn more about the collectives</a>
+          {false && <a href='#'>Learn more about the collectives</a> }
```

Comment:
It was to still have it there and make sure we know how to add it back quickly. 
Is that bothering?